scene/main/viewport.cpp

```diff
@@ -3406,15 +3406,17 @@ Variant Viewport::gui_get_drag_data() const {
 	return gui.drag_data;
 }
 
-PackedStringArray Viewport::get_configuration_warnings() const {
-	ERR_MAIN_THREAD_GUARD_V(PackedStringArray());
-	PackedStringArray warnings = Node::get_configuration_warnings();
+#ifdef TOOLS_ENABLED
+Array Viewport::get_configuration_info() const {
+	ERR_MAIN_THREAD_GUARD_V(Array());
+	Array warnings = Node::get_configuration_info();
 
 	if (size.x <= 1 || size.y <= 1) {
-		warnings.push_back(RTR("The Viewport size must be greater than or equal to 2 pixels on both dimensions to render anything."));
+		CONFIG_WARNING(RTR("The Viewport size must be greater than or equal to 2 pixels on both dimensions to render anything."));
```

Comment:
For `size` property.
Although looks like another dead warning. You can't set too small size.